server/utils/next_url.ts

```diff
@@ -24,3 +24,9 @@ export function composeNextUrlQeuryParam(request: KibanaRequest, basePath: strin
   const nextUrl = format(url);
   return stringify({ nextUrl });
 }
+
+export const validateNextUrl = (url: string | undefined): string | void => {
+  if (url && !url.startsWith('/')) {
```

Comment:
Could you verify if these 2 cases are handled:
http://localhost:5601/gyq/app/login?nextUrl=//opendistro.github.io
http://localhost:5601/gyq/app/login?nextUrl=%2F%2Fopendistro.github.io

Reference PR: 
https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/pull/463